src/stp/computations/PassComputations.cpp

```diff
@@ -16,17 +16,17 @@ namespace rtt::ai::stp::computations {
                            [&](const auto &bot) { return passTube.contains(bot->getPos()); });
     }
 
-    Vector2 PassComputations::determineBestPosForPass(
-            std::vector<computations::PositionComputations::PositionEvaluation> &positions) {
-        if (!positions.empty()) {
-            return std::max_element(positions.begin(), positions.end(),
-                                    [](PositionComputations::PositionEvaluation &left,
-                                       PositionComputations::PositionEvaluation &right) {
-                                        return left.score < right.score;
-                                    })->position;
-        } else {
-            RTT_DEBUG("There were no possible locations to pass to.");
-            return {};
-        }
-    }
+//    Vector2 PassComputations::determineBestPosForPass(
+//            std::vector<computations::PositionComputations::PositionEvaluation> &positions) {
+//        if (!positions.empty()) {
+//            return std::max_element(positions.begin(), positions.end(),
+//                                    [](PositionComputations::PositionEvaluation &left,
+//                                       PositionComputations::PositionEvaluation &right) {
+//                                        return left.score < right.score;
+//                                    })->position;
+//        } else {
+//            RTT_DEBUG("There were no possible locations to pass to.");
+//            return {};
+//        }
+//    }
```

Comment:
Why was this commented? can it be removed?

Reply:
Sorry, forgot the comment, The code cant work in this play and i needed the play for testing. The code iwll be used in a new play and therefore i didnt remove it just yet.